File: Semestr 1/TablicaChar/Kod_ASCII_w_trzech_kolumnach.cpp
/*autor: Pawel Sadowski*/
/* Program wypisuje wartrosc ASCII dla liczb calkowitych z zadanego zakresu w trzech kolumnach*/
#include <iostream>
using namespace std;

int main()
{
    int n_p,n_k,j=1;

    cin >> n_p >> n_k;

    for(int i=n_p; i <= n_k; i++)
    {
        cout << i << " - " << char(i);
        cout << '\t';
        if(j % 3 == 0)
            cout << endl;
        j++;
    }
    return 0;
}

File: Semestr 1/TablicaChar/Python/kod_ASCII_w_trzech_kolumnach.py
"""
autor: Paweł Sadowski
Program wypisuje wartrosc ASCII dla liczb calkowitych z zadanego zakresu w trzech kolumnach
"""
n_p = 0;
n_k = 0;
j = 1;

n_p = int(input("Podaj początkową liczbę: "))
n_k = int(input("Podaj końcową liczbę: "))

for i in range(n_p, n_k + 1):
    print(i, ' - ', chr(i), end = ' ')
    if j % 3 == 0:
        print();
    j += 1;